docs/runbooks/maintenance_mode/manage_maintenance.sh

```diff
@@ -34,28 +34,28 @@ function parse_args() {
           shift
           shift
           ;;
-          -m|--maintenace_mode)
-          MAINTENACE_MODE=True
+          -m|--maintenance_mode)
+          MAINTENANCE_MODE=True
           shift
           ;;
-          -d|--disable_maintenace_mode)
-          MAINTENACE_MODE=False
+          -d|--disable_maintenance_mode)
+          MAINTENANCE_MODE=False
           shift
           ;;
       esac
   done
 }
 
 function start() {
-  if [ $MAINTENACE_MODE = "True" ]
+  if [ $MAINTENANCE_MODE = "True" ]
   then
     enable_maintenance
   else
     disable_maintenance
   fi
 }
 
-MAINTENACE_MODE=False
+MAINTENAnCE_MODE=False
```

Comment:
I'm being very picky- the case is mixed here...

Reply:
![image](https://user-images.githubusercontent.com/6545556/76069761-44590900-5f8b-11ea-827b-1c4ee4e0ad0a.png)